Code/Tools/FBuild/FBuildCore/WorkerPool/JobQueue.cpp

```diff
@@ -203,10 +203,22 @@ bool JobQueue::HaveWorkersStopped() const
 
 // GetNumDistributableJobsAvailable
 //------------------------------------------------------------------------------
```

Comment:
This function is called in two places, and numJobsAvailable is not used in one case, and checked redundantly in another.
Additionally, I don't think there should be any nullptr entries in this array.
Maybe I'm missing something about your changes - can you provide some info please?